nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/document/DocumentModelJsonWriter.java

```diff
@@ -144,6 +148,12 @@ protected void writeEntityBody(DocumentModel doc, JsonGenerator jg) throws IOExc
         jg.writeStringField("state", doc.getRef() != null ? doc.getCurrentLifeCycleState() : null);
         jg.writeStringField("parentRef", doc.getParentRef() != null ? doc.getParentRef().toString() : null);
         jg.writeBooleanField("isCheckedOut", doc.isCheckedOut());
+        jg.writeBooleanField("isRecord", doc.isRecord());
```

Comment:
`jg.writeBooleanField("isRecord", doc.getRef() != null && doc.isRecord());` to avoid error on doc without session

Reply:
Fixed at `DocumentModel` level.